cmake/O2Dependencies.cmake

```diff
@@ -48,15 +48,19 @@ find_package(HEPMC)
 # PR #886 depending on some header files conflict with the IWYU setup
 # disable package for the moment
 #find_package(IWYU)
-find_package(DDS)
 
 find_package(Boost 1.59 COMPONENTS thread system timer program_options random filesystem chrono exception regex serialization log log_setup unit_test_framework date_time signals REQUIRED)
 # for the guideline support library
 include_directories(${MS_GSL_INCLUDE_DIR})
 
 find_package(AliRoot)
 find_package(FairRoot REQUIRED)
-find_package(FairMQ REQUIRED)
+find_package(FairMQInFairRoot)
+if(NOT FairMQInFairRoot_FOUND)
+  find_package(FairMQ REQUIRED)
+  find_package(FairLogger REQUIRED)
+endif()
+find_package(DDS)
```

Comment:
Two comments to how this works:

1. `FairLogger` is a transitive dependency from `FairRoot` **and** `FairMQ`.

2. I moved the `find_package(DDS)` after the `find_package(FairMQ)` because the latter will import a proper `FindDDS.cmake`, which was previously shipped by `FairRoot` and imported much earlier (DDS does not provide a CMake package so far, so we need a find module).